dictionaries/top-10000-project-gutenberg-words.json

```diff
@@ -5144,6 +5144,7 @@
 "SPHEUGS": "submission",
 "SPWREU": "entry",
 "RAEUFPL": "Rachel",
+"RAEU/KHEL": "Rachel",
```

Comment:
@paulfioravanti let's also remove the previous entry for Rachel

Reply:
Ah, whoops! That should have been a change, not an addition. I'll fix that :+1:

Reply:
Done ✅